apps/lend/src/store/createLoanSelfLiquidationSlice.ts

```diff
@@ -18,6 +18,10 @@ type SliceState = {
   liquidationAmt: string
 }
 
+export enum FormWarning {
+  NotEnoughCRVUSD = 'warning-not-enough-crvusd',
+}
```

Comment:
```ts
NotInLiquidation: 'warning-not-in-liquidation-mode'
```

Reply:
I am actually not using 'warning-not-in-liquidation-mode' anymore

Reply:
You already have a list of Warnings, find `ALERT_FORM_WARNING_KEYS`, convert it to a similar enum and use it.
```ts
enum AlertFormWarning { ... }

const messages: { [key: AlertFormWarningKey]: { message: string; alertType?: AlertType } } = {
      AlertFormWarning.FullRepaymentOnly: {
        message: t`Only full repayment is allowed when in soft-liquidation mode.`,
      },
```